Grbl_Esp32/src/Spindles/Spindle.h

```diff
@@ -26,10 +26,6 @@
 
 */
 
-#define SPINDLE_STATE_DISABLE 0   // Must be zero.
-#define SPINDLE_STATE_CW bit(4)   // matches PL_COND_FLAG_SPINDLE_CW
-#define SPINDLE_STATE_CCW bit(5)  // matches PL_COND_FLAG_SPINDLE_CCW
-
 #define SPINDLE_TYPE_NONE 0
```

Comment:
I am trying to restrict the scope of this patch to common code that mainly affects the parser, leaving the Spindles and Motors for another patch. I think it will be easier to test that way.